data.html

```diff
@@ -82,253 +85,139 @@
-							<td class="download_button">
-								<a href="https://s3.amazonaws.com/net-zero/2015-data-files/ElecEnergy.csv"><div class="button_div">Download</div></a>
-							</td>
-
-							<td>
-								<div class="clipboard_icon" data-toggle="popover" data-placement="left" data-content="https://s3.amazonaws.com/net-zero/2015-data-files/ElecEnergy.csv"><i class="fa fa-clipboard fa-2x" aria-hidden="true" id="ElecEnergy"></i></div>
-							</td>
-						</tr>
-
-						<tr>
-							<td>ElecPower</td>
-							<td>No Definition Provided</td>
-							<td class="download_button">
-								<a href="https://s3.amazonaws.com/net-zero/2015-data-files/ElecPower.csv"><div class="button_div">Download</div></a>
-							</td>
-
-							<td>
-								<div class="clipboard_icon" data-toggle="popover" data-placement="left" data-content="https://s3.amazonaws.com/net-zero/2015-data-files/ElecPower.csv"><i class="fa fa-clipboard fa-2x" aria-hidden="true" id="ElecPower"></i></div>
-							<td>
-						</tr>
-
-						<tr>
-							<td>ElecRPB</td>
-							<td>No Definition Provided</td>
-							<td class="download_button">
-								<a href="https://s3.amazonaws.com/net-zero/2015-data-files/ElecRPB.csv"><div class="button_div">Download</div></a>
-							</td>
-
-							<td>
-								<div class="clipboard_icon" data-toggle="popover" data-placement="left" data-content="https://s3.amazonaws.com/net-zero/2015-data-files/ElecRPB.csv"><i class="fa fa-clipboard fa-2x" aria-hidden="true" id="ElecRPB"></i></div>
-							</td>
-						</tr>
-
-						<tr>
-							<td>ElecTotal</td>
-							<td>No Definition Provided</td>
-							<td class="download_button">
-								<a href="https://s3.amazonaws.com/net-zero/2015-data-files/ElecTotal.csv"><div class="button_div">Download</div></a>
-							</td>
-
-							<td>
-								<div class="clipboard_icon" data-toggle="popover" data-placement="left" data-content="https://s3.amazonaws.com/net-zero/2015-data-files/ElecTotal.csv"><i class="fa fa-clipboard fa-2x" aria-hidden="true" id="ElecTotal"></i></div>
-							</td>
-						</tr>
-
-						<tr>
-							<td>Instr</td>
-							<td>Instrumentation</td>
-							<td class="download_button">
-								<a href="https://s3.amazonaws.com/net-zero/2015-data-files/Instr.csv"><div class="button_div">Download</div></a>
-							</td>
-
-							<td>
-								<div class="clipboard_icon" data-toggle="popover" data-placement="left" data-content="https://s3.amazonaws.com/net-zero/2015-data-files/Instr.csv"><i class="fa fa-clipboard fa-2x" aria-hidden="true" id="Instr"></i></div>
+								<a tabindex="0" class="clipboard_icon" role="button" data-toggle="popover" data-placement="bottom" data-trigger="focus" data-content="https://s3.amazonaws.com/nist-netzero/2015-data-files/Elec.csv"><i class="fa fa-clipboard fa-2x" aria-hidden="true"></i></a>
 							</td>
 						</tr>
 
 						<tr>
 							<td>All Subsystems</td>
 							<td>All Subsystem Data</td>
 							<td class="download_button">
```

Comment:
this button also needs the 'col_not_mobile' class. On mobile, this button still appears while all the other ones disappear